Wikipedia/Code/WKWebView+WMFWebViewControllerJavascript.swift

```diff
@@ -156,27 +156,30 @@ extension WKWebView {
         let readMoreItemCount = 3
         evaluateJavaScript("window.wmf.footerReadMore.add('\(title)', \(readMoreItemCount), 'pagelib_footer_container_readmore_pages', '\(proxyURL)', \(saveButtonTapHandler), \(titlesShownHandler), document);", completionHandler: nil)
     }
-    
-    public func wmf_enableCompatibilityTransformSupport(){
-        evaluateJavaScript("window.wmf.compatibility.enableSupport(document);", completionHandler: nil)
-    }
 
     public func wmf_classifyThemeElements(){
```

Comment:
I'm not sure where you call this in the browsing view but don't forget to do the same in the preview view. (BTW, think I'd like to consider merging bundle.js and preview.js later.)